config/locales/de/external/devise.yml

```diff
@@ -95,6 +95,7 @@ de:
     registrations:
       destroyed: Ihr Account wurde gelöscht. Wir hoffen, Sie bald wiederzusehen.
       edit:
+        api_key_usage: 'Geben Sie hier Ihren OpenAI API-Schlüssel ein, um verschiedene KI-Funktionen innerhalb der Plattform zu nutzen. Einen API-Schlüssel können Sie von der '
```

Comment:
If you include the Website here (and thereby complete the sentence), we could also fix the German sentence: There is a verb missing at the end of the sentence. Correct would be something like: `Einen API-Schlüssel können Sie auf der <a ...>Webseite von OpenAI</a> erstellen.`

Reply:
I am referring to [this comment](https://github.com/openHPI/codeharbor/pull/1496#discussion_r1672994562) regarding the inclusion of the website and otherwise just suggested another German locale and highlighted the link text (between the `<a>` tags).

Reply:
the <a> tag didn't work with the link:

`"Geben Sie hier Ihren OpenAI API-Schlüssel ein, um verschiedene KI-Funktionen innerhalb der Plattform zu nutzen. Einen API-Schlüssel können Sie auf der <a href='https://platform.openai.com/api-keys'>Website von OpenAI</a> erstellen."`

Am i doing it correctly?

Reply:
I was able to fix it :)

Reply:
I pushed the new changes